test/features/discord.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Discord Module
+
+  Scenario: Get List of Users
+    Given make a GET request to "/discord"
+    Then the response status code should be "200"
+    And the response should be "[]"
+
+
+  Scenario: Add a New User
+    Given make a POST request to "/discord"
```

Comment:
We will need to add a table so POST data can be added/changed here, but this can come in a future PR